src/map.rs

```diff
@@ -346,19 +346,18 @@ where
 #[cfg(test)]
 mod test {
     use super::*;
-    use serde::{Deserialize, Serialize};
-    use std::ops::Deref;
-
+    #[cfg(feature = "iterator")]
+    use crate::bound::Bounder;
+    use crate::int_key::IntKey;
     use cosmwasm_std::testing::MockStorage;
+    #[cfg(feature = "iterator")]
     use cosmwasm_std::to_json_binary;
+    #[cfg(feature = "iterator")]
```

Comment:
Putting all these exports and feature-gated tests in an `iterator` submodule could also be an idea!